float32[] center_line_x
float32[] center_line_y
float32[] center_line_yaw
bool lane_mode
int8 speed_mode